issue-rules.yml

```diff
@@ -59,17 +59,174 @@ rules:
   assign: ['madhurig']     
 # Area: Release
 - valueFor: '**Enter Task Name**'
-  contains: 'Azure'
```

Comment:
At first I wondered why just not keep the general Azure rule but I see now that you want to optimize for assign.  Nice!

Reply:
Also take a look at the fall back rules (text contains).  However, I think this subset of changes are good.  From here out, I think you take it on a case by case basis of "why was this issue not routed correctly"

Reply:
Sure. Make sense.